### PR TITLE
Us2_1_t3: Implement API for generating payment QR code 

### PR DESCRIPTION
**Pull request by FaiiFaiipuipui**
- created API for generating with qrcode (doc is in transactions controller)
- created PUML and its PNG export of Backend Site Map
- exported schema PUML to PNG

### CODE REVIEW

File: models/DatabaseSchema.png
Comment (Yiwahpsp) [inline models/DatabaseSchema.png:1]: Can a campground has many appointments

File: utils/createPromptpayQR.js
Comment (Yiwahpsp) [inline utils/createPromptpayQR.js:1]: Geng makk

**Review by Yiwahpsp — APPROVED** [review]
Look Good

**Review by BoondaPanich — APPROVED** [review]
Sood yoddd👏

**Review by Yiwahpsp — APPROVED** [review]
lgtm

**Review by BoondaPanich — APPROVED** [review]
👍👍👍

**Review by tippaphanun — APPROVED** [review]

**Pull request merged**